src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -144,15 +161,25 @@ public abstract class DiseaseAnnotation extends Annotation {
 	@JsonView({View.FieldsOnly.class, View.ForPublic.class})
 	private VocabularyTerm geneticSex;
 
-	@IndexedEmbedded(includePaths = {"sourceOrganization.abbreviation", "sourceOrganization.fullName", "sourceOrganization.shortName", "crossReference.displayName", "crossReference.referencedCurie",
-		"sourceOrganization.abbreviation_keyword", "sourceOrganization.fullName_keyword", "sourceOrganization.shortName_keyword", "crossReference.displayName_keyword", "crossReference.referencedCurie_keyword"})
+	@IndexedEmbedded(includePaths = {
+		"abbreviation", "fullName", "shortName",
+		"abbreviation_keyword", "fullName_keyword", "shortName_keyword"
+	})
 	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
 	@ManyToOne
-	@JsonView({View.FieldsOnly.class, View.ForPublic.class})
-	private DataProvider secondaryDataProvider;
+	@Fetch(FetchMode.SELECT)
+	@JsonView({ View.FieldsOnly.class })
```

Comment:
Again ForPublic was removed... this may cause issue on the public side.

Reply:
Replaced